pkg/auth/cell.go

```diff
@@ -45,23 +45,26 @@ var Cell = cell.Module(
 		newAlwaysFailAuthHandler,
 	),
 	cell.Config(config{
-		MeshAuthEnabled:    true,
-		MeshAuthQueueSize:  1024,
-		MeshAuthGCInterval: 5 * time.Minute,
+		MeshAuthEnabled:           true,
+		MeshAuthQueueSize:         1024,
+		MeshAuthGCInterval:        5 * time.Minute,
+		MeshAuthSignalBackoffTime: 1 * time.Second, // this default is based on the default TCP retransmission timeout
 	}),
 	cell.Config(MutualAuthConfig{}),
 )
 
 type config struct {
-	MeshAuthEnabled    bool
-	MeshAuthQueueSize  int
-	MeshAuthGCInterval time.Duration
+	MeshAuthEnabled           bool
+	MeshAuthQueueSize         int
+	MeshAuthGCInterval        time.Duration
+	MeshAuthSignalBackoffTime time.Duration
 }
 
 func (r config) Flags(flags *pflag.FlagSet) {
 	flags.Bool("mesh-auth-enabled", r.MeshAuthEnabled, "Enable authentication processing & garbage collection (beta)")
 	flags.Int("mesh-auth-queue-size", r.MeshAuthQueueSize, "Queue size for the auth manager")
 	flags.Duration("mesh-auth-gc-interval", r.MeshAuthGCInterval, "Interval in which auth entries are attempted to be garbage collected")
+	flags.Duration("mesh-auth-signal-backoff-time", r.MeshAuthSignalBackoffTime, "Time to wait betweeen two authentication required signals in case of a cache mismatch")
```

Comment:
nit, since "time" is a bit ambiguous whether it means a specific point in time or a duration:
```suggestion
	flags.Duration("mesh-auth-signal-backoff-duration", r.MeshAuthSignalBackoffTime, "Time to wait betweeen two authentication required signals in case of a cache mismatch")
```

I also wonder if we should mark this as hidden. Most users are not going to need to customize it, but I do think it's worth having here in case it turns out to be useful to modify.

Reply:
Related to the "hidden" comment, I see that CI failed specifically because the docs weren't yet update for this flag. I guess if we hide it then no docs updates are necessary, but either way the PR needs to fix up that CI failure.

Reply:
I marked it as hidden so the documentation will be correct